csss-site/src/events/templates/events/mountainmadness.github.io/index.html

```diff
@@ -0,0 +1,266 @@
+<!DOCTYPE html>
```

Comment:
this should be under the location `csss-site/src/events/templates/events/index.html`